src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -40,73 +38,73 @@ interface SVGData {
  * @param identifier Landscape Identifier
  * @param onItemClick Handler for our label click
  */
-const Map: React.FC<Props> = () => {
+const Map: React.FC<Props> = ({ setSidebarContent, setFindFunction, setPageTitle }) => {
```

Comment:
This doc is outdated (onItemClick does not appear in function signature) and "choosen" is misspelled

Reply:
fixed